tests/utils/test_dag_processing.py

```diff
@@ -85,6 +87,23 @@ class TestDagFileProcessorManager(unittest.TestCase):
     def setUp(self):
         clear_db_runs()
 
+    def run_processor_manager_one_loop(self, manager, parent_pipe):
+        if not manager._async_mode:
+            parent_pipe.send(DagParsingSignal.AGENT_RUN_ONCE)
+
+        results = []
+
+        while True:  # pylint: disable=too-many-nested-blocks
+            manager._run_parsing_loop()
+
+            while parent_pipe.poll(timeout=0.01):
+                obj = parent_pipe.recv()
+                if isinstance(obj, DagParsingStat):
+                    if obj.done:
+                        return results
+                    continue
+                results.append(obj)
```

Comment:
```suggestion
                if not isinstance(obj, DagParsingStat):
                    continue
                elif obj.done:
                    return results
                results.append(obj)
```
This may please pylint

Reply:
Yes okay, I was being lazy :)
Would please plyint, but you've inverted one case.

```python
                if not isinstance(obj, DagParsingStat):
                    results.append(obj)
                elif obj.done:
                    return results
```

Reply:
Done.